test/grbl.js

```diff
@@ -200,6 +200,21 @@ test('GrblLineParserResultParserState', (t) => {
         runner.parse(line);
     });
 
+    test('#3', (t) => {
+        const runner = new GrblRunner();
+        runner.on('parserstate', ({ raw, ...parserstate }) => {
+            t.equal(raw, '[T2 M6.]');
```

Comment:
`[T2 M6.]` is not a valid Grbl parser state message. The above line can be changed to `[GC:G0 G54 G17 G21 G90 G94 M5 M9 T2 F0 S0]` to validate the tool number with a value of 2.